### PR TITLE
Revert "sql: GA load data logical mode and add experimental physical mode"

### PR DESCRIPTION
**Pull request by Oreoxmt**
Reverts pingcap/docs-cn#13708

The following features are removed from both v7.0 and v7.1:

- Support importing data from SQL files and Parquet
- Support setting `bath_size` which specifies the number of rows inserted into the target table per batch, to improve write performance.
- Support setting `detached` to allow the job to run in the background.
- Support using `SHOW LOAD DATA` and `DROP LOAD DATA` to manage jobs.

docs PR: https://github.com/pingcap/docs/pull/13606

### CODE REVIEW

File: sql-statements/sql-statement-operate-load-data-job.md
Comment (Oreoxmt) [inline sql-statements/sql-statement-operate-load-data-job.md:1]: Please remove `sql-statements/sql-statement-operate-load-data-job.md` and `sql-statements/sql-statement-show-load-data.md` in TOC.md

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -8,52 +8,28 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-load-data/','/docs-cn/dev/r
...
-- 支持导入压缩的 `DELIMITED DATA` 和 `SQL FILE` 数据文件
-- 逻辑导入支持并发导入
-- 支持通过 `CharsetOpt` 来指定数据文件的编码格式
-- `LOAD DATA` 集成 TiDB Lightning 的物理导入模式 (Physical Import Mode)。该模式不经过 SQL 接口，而是直接将数据以键值对的形式插入 TiKV 节点，是一种高效且快速的导入模式。
+- 新增参数 `FIELDS DEFINED NULL BY`
 
 > **警告：**
 >
-> v7.1.0 新增的并发导入和物理导入模式为实验特性，不建议在生产环境中使用。
+> 新增的能力和参数为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-load-data.md:18]:
```suggestion
> v7.0.0 新增支持从 S3、GCS 导入数据和 `FIELDS DEFINED NULL BY` 参数为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
```

File: sql-statements/sql-statement-operate-load-data-job.md
Comment (D3Hunter) [reply sql-statements/sql-statement-operate-load-data-job.md:1]: done

File: releases/release-7.1.0.md [hunk 2]
```diff
@@ -151,27 +151,11 @@ TiDB 版本：7.1.0 (upcoming)
     在 TiDB v7.1.0 中，以下 `LOAD DATA` 功能 GA：
 
     - 支持从 S3、GCS 导入数据。
```
Comment (Frank945946) [inline releases/release-7.1.0.md:153]:
```suggestion

```

File: releases/release-7.1.0.md [hunk 3]
```diff
@@ -151,27 +151,11 @@ TiDB 版本：7.1.0 (upcoming)
...
-    - 支持导入 Parquet 格式的数据。
     - 支持解析源文件中的下列字符集：`ascii`、`latin1`、`binary`、`gbk` 和 `utf8mbd`
     - 支持设置 `FIELDS DEFINED NULL BY` 将源文件中的指定值转换为 `NULL` 并写入目标表。
```
Comment (Frank945946) [inline releases/release-7.1.0.md:155]:
```suggestion
```

File: releases/release-7.0.0.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ TiDB 版本：7.0.0
   <tr>
     <td rowspan="2">数据库管理与可观测性<br/></td>
     <td>TiDB 通过 <a href="https://docs.pingcap.com/zh/tidb/v7.0/sql-statement-load-data" target="_blank"><code>LOAD DATA</code> 语句集成 TiDB Lightning</a>（实验特性）</td>
-    <td>集成 TiDB Lightning 的逻辑导入模式使 <code>LOAD DATA</code> 语句更加强大，例如支持从 S3/GCS 导入数据、支持任务管理等。</td>
+    <td>集成 TiDB Lightning 的逻辑导入模式使 <code>LOAD DATA</code> 语句更加强大，例如支持从 S3/GCS 导入数据。</td>
```
Comment (Oreoxmt) [inline releases/release-7.0.0.md:60]: @Frank945946 PTAL

File: releases/release-7.0.0.md [hunk 5]
```diff
@@ -264,9 +264,7 @@ TiDB 版本：7.0.0
```
Comment (Oreoxmt) [inline releases/release-7.0.0.md:264]: @Frank945946 PTAL

```
* `LOAD DATA` 语句集成 TiDB Lightning，你可以使用 `LOAD DATA` 语句完成原先需要使用 TiDB Lightning 才能完成的数据导入任务（实验特性）[#40499](https://github.com/pingcap/tidb/issues/40499) @[lance6716](https://github.com/lance6716)
    在集成 TiDB Lightning 之前，`LOAD DATA` 语句只能用于导入客户端的数据文件，如果你需要从云存储导入数据，不得不借助 TiDB Lightning 来实现。但是单独部署 TiDB Lightning 又会带来额外的部署成本和管理成本。将 TiDB Lightning 逻辑导入能力 (Logical Import Mode) 集成到 `LOAD DATA` 语句后，不仅可以省去 TiDB Lightning 的部署和管理成本，还可以借助 TiDB Lightning 的功能极大扩展 `LOAD DATA` 语句的能力。部分扩展的功能举例说明如下：

    - 支持从 Amazon S3 和 Google Cloud Storage 导入数据到 TiDB，且支持使用通配符一次性匹配多个源文件导入到 TiDB。
    - 支持 `DEFINED NULL BY` 来定义 null。
    - 支持 CSV、TSV 格式的源文件。

```

Comment (Frank945946) [reply releases/release-7.0.0.md:264]: 我在你的基础上稍微改了一下，去掉了集成 Lightning 的描述。因为以后不会集成 Lightning 更多功能了，因此尽量不提 Lightning了。

File: sql-statements/sql-statement-load-data.md [hunk 6]
```diff
@@ -234,7 +119,6 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
 >
 > - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。
 > - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
-> - 从 TiDB v7.0.0 开始，批量提交的行数由 `LOAD DATA` 语句的 `WITH batch_size=<number>` 参数控制，默认 1000 行提交一次。
 > - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。
```
[the remark this answers is not in the document]
Comment (Frank945946) [reply sql-statements/sql-statement-load-data.md:122]: LGTM

**Review by Frank945946 — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**